sw/device/lib/dif/dif_plic.c

```diff
@@ -16,7 +16,7 @@
 // If either of these static assertions fail, then the assumptions in this DIF
 // implementation should be revisited. In particular, `kPlicTargets`
 // may need updating,
-_Static_assert(RV_PLIC_PARAM_NUMSRC == 99,
+_Static_assert(RV_PLIC_PARAM_NUMSRC == 123,
```

Comment:
wow, does the UART add so many different interrupts? do we have plans to consolidate some of them?